back/app/services/side_fx_user_service.rb

```diff
@@ -32,6 +32,15 @@ def after_update(user, current_user)
     SendConfirmationCode.call(user: user) if user.should_send_confirmation_email?
   end
 
+  def before_destroy(user, _current_user)
+    # Remove all reactions by the user on initiatives that are in the active voting phase ('proposed' status).
+    # Ensures 1 reaction per verified user per initiative, if verification (by a single method) is required to react,
+    # since it should not be possible to create more than one account (concurrently) for the same verification.
+    # We do this before the user is destroyed, as we nullify user_id in reactions when a user is destroyed.
+    proposed_initiatives_reactions = user.reactions.where(reactable_id: [Initiative.proposed])
```

Comment:
Two minor changes
```suggestion
    proposed_initiatives_reactions = user.reactions.where(reactable_id: Initiative.proposed, reactable_type: Initiative)
```

Reply:
Should we include `review_pending` and `changes_requested` here?

A user creates a proposal. It has the status `review_pending`. The first vote is cast automatically. The user deletes their account. The vote is not deleted. The user creates a new account and votes again.

Reply:
Is there any reason not to delete votes of ideas? I think on the call we more or less agreed on it. But anyway, better to discuss with Simon F.

Reply:
Good point about review_pending and status requested.

Not sure we agreed it for ideas also. I thought we decided it's probably best to start with proposals and think about ideas later.

Reply:
> Is there any reason not to delete votes of ideas?

Agreed in meeting today (Koen, Alex & I) that this could be more complex and move into scope creep. We will instead document the differences between ideas and proposals, including this one.